bridges/modules/relayers/src/lib.rs

```diff
@@ -83,6 +101,22 @@ pub mod pallet {
 		type WeightInfo: WeightInfoExt;
 		/// Lane identifier type.
 		type LaneId: LaneIdType + Send + Sync;
+		/// AssetHub parachain ID
+		type AssetHubParaId: Get<u32>;
+		type InboundQueuePalletInstance: Get<u8>;
+		/// Ethereum network ID including the chain ID
+		type EthereumNetwork: Get<NetworkId>;
+		/// Message relayers are rewarded with this asset
+		type WethAddress: Get<H160>;
+		/// XCM message sender
+		type XcmSender: SendXcm;
+		/// To withdraw and deposit an asset.
+		type AssetTransactor: TransactAsset;
+		type AssetHubXCMFee: Get<u128>;
+		type Token: Mutate<Self::AccountId> + Inspect<Self::AccountId>;
+		/// TreasuryAccount to collect fees
+		#[pallet::constant]
+		type TreasuryAccount: Get<Self::AccountId>;
```

Comment:
We need to refactor `pallet-bridge-relayers` to support multiple rewards,I will prepare issue with step-by-step migration: https://github.com/paritytech/polkadot-sdk/issues/7272

```suggestion
```

Reply:
@claravanstaden I've prepared changes https://github.com/paritytech/polkadot-sdk/pull/7492, where on the top of that you should be able to achieve register/claim_rewards for Snowbridge